### PR TITLE
Use Node.js v14 features

### PR DESCRIPTION
**Pull request by tlfn**
**Please describe the changes this PR makes and why it should be merged:**
By using Node v14, we can use new features such as Optional chaining, Nullish Coalescing, ...
And using those features make the code more readable, and cleaner.

**Status**

- [ ] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes
- [ ] Documentation has been added/modified, or there is nothing to change (docs/mods.json)

**Semantic versioning classification:**

- [ ] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by Fisheiyy** [comment]
have you tested anything? also im not sure if its possible to migrate to a newer version of node without having DBM itself be updated to a newer version of node

**Comment by TheMonDon** [comment]
> have you tested anything? also im not sure if its possible to migrate to a newer version of node without having DBM itself be updated to a newer version of node

It's a draft PR, this will be merged after DBM update and discord.js v13 comes out. Not tested yet just preliminary code.

File: actions/check_if_member_MOD.js [hunk 1]
```diff
@@ -132,10 +132,10 @@ module.exports = {
         result = member.kickable
         break
       case 3:
-        result = !!this.dest(member.voice, 'speaking')
+        result = member.voice?.speaking ?? false
```
Comment (almeidx) [inline actions/check_if_member_MOD.js:135]:
```suggestion
        result = !!member.voice?.speaking
```

File: actions/check_if_member_MOD.js [hunk 2]
```diff
@@ -132,10 +132,10 @@ module.exports = {
...
         break
       case 4:
-        result = !!this.dest(member.voice, 'channel')
+        result = member.voice?.channel ?? false
```
Comment (almeidx) [inline actions/check_if_member_MOD.js:138]:
```suggestion
        result = !!member.voice?.channel
```

File: actions/check_if_member_MOD.js [hunk 3]
```diff
@@ -144,10 +144,10 @@ module.exports = {
         result = member.id === Files.data.settings.ownerId
         break
       case 7:
-        result = !!this.dest(member.voice, 'mute')
+        result = member.voice?.mute ?? false
```
Comment (almeidx) [inline actions/check_if_member_MOD.js:147]:
```suggestion
        result = !!member.voice?.mute
```

File: actions/check_if_member_MOD.js [hunk 4]
```diff
@@ -144,10 +144,10 @@ module.exports = {
...
         break
       case 8:
-        result = !!this.dest(member.voice, 'deaf')
+        result = member.voice?.deaf ?? false
```
Comment (almeidx) [inline actions/check_if_member_MOD.js:150]:
```suggestion
        result = !!member.voice?.deaf
```

File: actions/check_if_member_has_role_MOD.js [hunk 5]
```diff
@@ -116,15 +116,9 @@ module.exports = {
...
-        result = member.every(function (mem) {
-          if (this.dest(mem, 'roles', 'cache')) {
-            return mem.roles.cache.has(role.id)
-          } else {
-            return false
-          }
-        })
-      } else if (this.dest(member, 'roles', 'cache')) {
-        result = member.roles.cache.has(role.id)
+        result = member.every((mem) => mem?.roles?.cache?.has?.(role.id) === true)
```
Comment (almeidx) [inline actions/check_if_member_has_role_MOD.js:119]:
```suggestion
        result = member.every((mem) => mem?.roles?.cache.has(role.id))
```

File: actions/check_if_member_has_role_MOD.js [hunk 6]
```diff
@@ -116,15 +116,9 @@ module.exports = {
...
+      } else {
+        result = member?.roles?.cache?.has?.(role.id) === true
```
Comment (almeidx) [inline actions/check_if_member_has_role_MOD.js:121]:
```suggestion
        result = !!member?.roles?.cache.has(role.id)
```

File: actions/check_if_user_reacted_MOD.js [hunk 7]
```diff
@@ -114,13 +114,9 @@ module.exports = {
...
     let result
     if (Array.isArray(member)) {
-      result = member.every(user => {
-        if (this.dest(reaction, 'users', 'cache') && user) {
-          return reaction.users.cache.has(user.id)
-        }
-      })
-    } else if (this.dest(reaction, 'users', 'cache') && member) {
-      result = reaction.users.cache.has(member.id)
+      result = member.every((user) => reaction?.users?.cache?.has?.(user.id) === true)
```
Comment (almeidx) [inline actions/check_if_user_reacted_MOD.js:117]:
```suggestion
      result = member.every((user) => reaction?.users?.cache.has(user.id))
```

File: actions/check_if_user_reacted_MOD.js [hunk 8]
```diff
@@ -114,13 +114,9 @@ module.exports = {
...
+    } else {
+      result = reaction?.users?.cache?.has?.(member.id) === true
```
Comment (almeidx) [inline actions/check_if_user_reacted_MOD.js:119]:
```suggestion
      result = !!reaction?.users?.cache.has(member.id)
```

File: actions/control_permissons_MOD.js [hunk 9]
```diff
@@ -146,26 +146,26 @@ module.exports = {
 
     permsArray.forEach((perms) => {
       if (data[perms] === 'Allow') {
-        if (!permissions.allow || !permissions.allow.has(perms)) {
+        if (!permissions.allow?.has?.(perms) !== true) {
```
Comment (almeidx) [inline actions/control_permissons_MOD.js:149]:
```suggestion
        if (!permissions.allow?.has(perms)) {
```

File: actions/control_permissons_MOD.js [hunk 10]
```diff
@@ -146,26 +146,26 @@ module.exports = {
...
           if (!permissions.allow) permissions.allow = new Permissions()
           permissions.allow.add(perms)
         }
-        if (permissions.disallow && permissions.disallow.has(perms)) permissions.disallow.remove(perms)
-        if (permissions.inherit && permissions.inherit.includes(perms)) permissions.inherit.splice(permissions.inherit.indexOf(perms), 1)
+        if (permissions.disallow?.has?.(perms) === true) permissions.disallow.remove(perms)
```
Comment (almeidx) [inline actions/control_permissons_MOD.js:153]:
```suggestion
        if (permissions.disallow?.has(perms)) permissions.disallow.remove(perms)
```

File: actions/control_permissons_MOD.js [hunk 11]
```diff
@@ -146,26 +146,26 @@ module.exports = {
...
+        if (permissions.inherit?.includes?.(perms) === true) permissions.inherit.splice(permissions.inherit.indexOf(perms), 1)
```
Comment (almeidx) [inline actions/control_permissons_MOD.js:154]:
```suggestion
        if (permissions.inherit?.includes(perms)) permissions.inherit.splice(permissions.inherit.indexOf(perms), 1)
```

File: actions/control_permissons_MOD.js [hunk 12]
```diff
@@ -146,26 +146,26 @@ module.exports = {
...
       } else if (data[perms] === 'Disallow') {
-        if (!permissions.disallow || !permissions.disallow.has(perms)) {
+        if (permissions.disallow?.has?.(perms) !== true) {
```
Comment (almeidx) [inline actions/control_permissons_MOD.js:156]:
```suggestion
        if (!permissions.disallow?.has(perms)) {
```

File: actions/control_permissons_MOD.js [hunk 13]
```diff
@@ -146,26 +146,26 @@ module.exports = {
...
           if (!permissions.disallow) permissions.disallow = new Permissions()
           permissions.disallow.add(perms)
         }
-        if (permissions.allow && permissions.allow.has(perms)) permissions.allow.remove(perms)
-        if (permissions.inherit && permissions.inherit.includes(perms)) permissions.inherit.splice(permissions.inherit.indexOf(perms), 1)
+        if (permissions.allow?.has?.(perms) === true) permissions.allow.remove(perms)
```
Comment (almeidx) [inline actions/control_permissons_MOD.js:160]:
```suggestion
        if (permissions.allow?.has(perms)) permissions.allow.remove(perms)
```

File: actions/control_permissons_MOD.js [hunk 14]
```diff
@@ -146,26 +146,26 @@ module.exports = {
...
+        if (permissions.inherit?.includes?.(perms) === true) permissions.inherit.splice(permissions.inherit.indexOf(perms), 1)
```
Comment (almeidx) [inline actions/control_permissons_MOD.js:161]:
```suggestion
        if (permissions.inherit?.includes(perms)) permissions.inherit.splice(permissions.inherit.indexOf(perms), 1)
```

File: actions/control_permissons_MOD.js [hunk 15]
```diff
@@ -146,26 +146,26 @@ module.exports = {
...
       } else if (data[perms] === 'Inherit') {
-        if (!permissions.inherit || !permissions.inherit.has(perms)) {
+        if (permissions.inherit?.has?.(perms) !== true) {
```
Comment (almeidx) [inline actions/control_permissons_MOD.js:163]:
```suggestion
        if (!permissions.inherit?.has(perms)) {
```

File: actions/control_permissons_MOD.js [hunk 16]
```diff
@@ -146,26 +146,26 @@ module.exports = {
...
           if (!permissions.inherit) permissions.inherit = []
           permissions.inherit.push(perms)
         }
-        if (permissions.disallow.has(perms)) permissions.disallow.remove(perms)
-        if (permissions.allow.has(perms)) permissions.allow.remove(perms)
+        if (permissions.disallow?.has?.(perms) === true) permissions.disallow.remove(perms)
```
Comment (almeidx) [inline actions/control_permissons_MOD.js:167]:
```suggestion
        if (permissions.disallow?.has(perms)) permissions.disallow.remove(perms)
```

File: actions/control_permissons_MOD.js [hunk 17]
```diff
@@ -146,26 +146,26 @@ module.exports = {
...
+        if (permissions.allow?.has?.(perms) === true) permissions.allow.remove(perms)
```
Comment (almeidx) [inline actions/control_permissons_MOD.js:168]:
```suggestion
        if (permissions.allow?.has(perms)) permissions.allow.remove(perms)
```

File: actions/create_animated_emoji_MOD.js [hunk 18]
```diff
@@ -66,8 +66,7 @@ module.exports = {
   action (cache) {
     const data = cache.actions[cache.index]
     const { server } = cache
-
-    if (!this.dest(server, 'emojis', 'create')) return this.callnextAction(cache)
+    if (server?.emojis?.create === undefined) return this.callnextAction(cache)
```
Comment (almeidx) [inline actions/create_animated_emoji_MOD.js:69]:
```suggestion
    if (!server) return this.callNextAction(cache)
```

File: actions/create_category_MOD.js [hunk 19]
```diff
@@ -44,7 +44,7 @@ Name:<br>
   action (cache) {
     const data = cache.actions[cache.index]
     const { server } = cache
-    if (!this.dest(server, 'channels', 'create')) return this.callnextAction(cache)
+    if (server?.channels?.create === undefined) return this.callnextAction(cache)
```
Comment (almeidx) [inline actions/create_category_MOD.js:47]:
```suggestion
    if (!server) return this.callNextAction(cache)
```

File: actions/delete_bulk_messages_MOD.js [hunk 20]
```diff
@@ -7,7 +7,7 @@ module.exports = {
 
   subtitle (data) {
     const channels = ['Same Channel', 'Mentioned Channel', '1st Server Channel', 'Temp Variable', 'Server Variable', 'Global Variable']
-    return `Delete ${data.count} messages from ${channels[parseInt(data.channel)] || 'Nothing'}`
+    return `Delete ${data.count} messages from ${channels[parseInt(data.channel)] ?? 'Nothing'}`
```
Comment (almeidx) [inline actions/delete_bulk_messages_MOD.js:10]:
```suggestion
    return `Delete ${data.count} messages from ${channels[parseInt(data.channel)] ?? 'nowhere'}`
```

File: actions/delete_bulk_messages_MOD.js [hunk 21]
```diff
@@ -132,7 +132,10 @@ module.exports = {
   },
 
   init () {
-    const { glob, document } = this
+    const {
+      glob,
+      document
+    } = this
```
Comment (almeidx) [inline actions/delete_bulk_messages_MOD.js:138]:
```suggestion
    const { glob, document } = this
```

File: actions/delete_bulk_messages_MOD.js [hunk 22]
```diff
@@ -217,78 +220,77 @@ module.exports = {
...
-              return result
-            })
-          }
-          if (Con5 !== '0') {
-            filtered = (filtered || messages).filter(
-              (e) => Con5 === '1' ? e.attachments.size === 0 : e.attachments.size !== 0
-            )
-          }
-          messagesFound = messagesFound.concat(filtered)
+    if (source?.messages?.fetch === undefined) return this.callNextAction(cache)
```
Comment (almeidx) [inline actions/delete_bulk_messages_MOD.js:223]:
```suggestion
    if (!source) return this.callNextAction(cache)

```

File: actions/delete_bulk_messages_MOD.js [hunk 23]
```diff
@@ -217,78 +220,77 @@ module.exports = {
...
+        if (Con3) filtered = (filtered ?? messages).filter((e) => e.content.includes(Con3))
+        if (Con4) {
+          filtered = (filtered ?? messages).filter(() => {
+            let result = false
+            try {
+              result = !!eval(Con4)
+            } catch {
+            }
+            return result
+          })
```
Comment (almeidx) [inline actions/delete_bulk_messages_MOD.js:263]:
```suggestion
          filtered = (filtered ?? messages).filter(() => {
            try {
              return !!eval(Con4)
            } catch {
              return false
            }
          })
```

File: actions/delete_bulk_messages_MOD.js [hunk 24]
```diff
@@ -217,78 +220,77 @@ module.exports = {
...
+        messagesFound = messagesFound.concat(filtered)
+      }
+      if (messagesFound.array) messagesFound = messagesFound.array()
+      if (messagesFound.length > limit) {
+        messagesFound.splice(limit)
+      }
+      const deleted = await source.bulkDelete(messagesFound)
+      const storage = parseInt(data.storage)
+      if (storage !== 0 && deleted) {
+        let result = deleted.array ? deleted.array() : deleted
```
Comment (almeidx) [inline actions/delete_bulk_messages_MOD.js:279]:
```suggestion
        let result = Array.isArray(deleted) ? deleted : deleted.array()
```

File: actions/delete_bulk_messages_MOD.js [hunk 25]
```diff
@@ -217,78 +220,77 @@ module.exports = {
...
+        this.executeResults(false, data, cache)
+      } else {
+        this.displayError(data, cache, err)
+      }
     }
   },
 
-  mod () {}
+  mod () {
+  }
```
Comment (almeidx) [inline actions/delete_bulk_messages_MOD.js:295]:
```suggestion
  mod () {}
```

File: actions/find_category_MOD.js [hunk 26]
```diff
@@ -91,7 +91,7 @@ module.exports = {
 
   action (cache) {
     const { server } = cache
-    if (!server || !server.channels) return this.callNextAction(cache)
+    if (server?.channels === undefined) return this.callNextAction(cache)
```
Comment (almeidx) [inline actions/find_category_MOD.js:94]:
```suggestion
    if (!server) return this.callNextAction(cache)
```

File: actions/find_member.js [hunk 27]
```diff
@@ -117,7 +117,7 @@ module.exports = {
 
   async action (cache) {
     const { server } = cache
-    if (!server || !server.members) return this.callNextAction(cache)
+    if (server?.members === undefined) return this.callNextAction(cache)
```
Comment (almeidx) [inline actions/find_member.js:120]:
```suggestion
    if (!server) return this.callNextAction(cache)
```

File: actions/remove_message_reactions_MOD.js [hunk 28]
```diff
@@ -36,13 +36,13 @@ module.exports = {
...
-      message.reactions.removeAll().then(() => {
-        this.callNextAction(cache)
-      }).catch(this.displayError.bind(this, data, cache))
+      this.callListFunc(message.map((m) => m.reactions), 'removeAll', [])
+        .then(() => this.callNextAction(cache))
+    } else {
+      // eslint-disable-next-line no-unused-expressions
+      message?.reactions?.removeAll?.()
+        ?.then?.(() => this.callNextAction(cache))
+        ?.catch?.(this.displayError.bind(this, data, cache))
```
Comment (almeidx) [inline actions/remove_message_reactions_MOD.js:45]:
```suggestion
      message?.reactions.removeAll()
        .then(() => this.callNextAction(cache))
        .catch(this.displayError.bind(this, data, cache))
```

File: actions/store_game_info_MOD.js [hunk 29]
```diff
@@ -147,7 +147,7 @@ module.exports = {
     const varName = this.evalMessage(data.varName, cache)
     const mem = this.getMember(member, varName, cache)
 
-    if (!mem || !mem.presence.activities[0]) return this.callNextAction(cache)
+    if (mem.presence?.activities?.[0] === undefined) return this.callNextAction(cache)
```
Comment (almeidx) [inline actions/store_game_info_MOD.js:150]:
```suggestion
    if (!mem?.presence?.activities[0]) return this.callNextAction(cache)
```

File: actions/store_user_info_MOD.js [hunk 30]
```diff
@@ -155,16 +162,12 @@ module.exports = {
         result = user.lastMessageID
         break
       case 7: // User Activities
-        if (this.dest(user.presence, 'activities')) {
-          const status = user.presence.activities.filter((s) => s.type !== 'CUSTOM_STATUS')
-          result = status && this.dest(status[0], 'name')
-        }
+        result = user.presence?.activities?.filter?.((s) => s?.type !== 'CUSTOM_STATUS')
```
Comment (almeidx) [inline actions/store_user_info_MOD.js:165]:
```suggestion
        result = user.presence?.activities.filter((s) => s.type !== 'CUSTOM_STATUS')
```

File: actions/store_user_info_MOD.js [hunk 31]
```diff
@@ -155,16 +162,12 @@ module.exports = {
...
+        result = result?.[0]?.name
         break
       case 8: // User Custom Status
-        if (this.dest(user.presence, 'activities')) {
-          const status = user.presence.activities.filter((s) => this.dest(s, 'type') === 'CUSTOM_STATUS')
-          result = status && this.dest(status[0], 'state')
-        }
+        result = user.presence?.activities?.filter?.((s) => s?.type === 'CUSTOM_STATUS')
```
Comment (almeidx) [inline actions/store_user_info_MOD.js:169]:
```suggestion
        result = user.presence?.activities.filter((s) => s.type === 'CUSTOM_STATUS')
```

File: actions/store_user_info_MOD.js [hunk 32]
```diff
@@ -179,12 +182,11 @@ module.exports = {
...
       case 13: // User Flags
-        const flags = user.flags
-        result = flags && flags.toArray()
+        result = user.flags?.toArray() ?? []
         break
       case 14: // User Status
-        const status = this.dest(user.presence, 'clientStatus')
-        result = status && Object.keys(status)
+        const status = user.presence?.clientStatus
+        result = Object.keys(status ?? {})
```
Comment (almeidx) [inline actions/store_user_info_MOD.js:189]:
```suggestion
        result = Object.keys(user.presence?.clientStatus ?? {})
```

**Review by almeidx — CHANGES_REQUESTED** [review]

File: actions/store_user_info_MOD.js (hunk 31 above)
Comment (tlfn) [reply actions/store_user_info_MOD.js:169]: This may cause an issue

File: actions/store_user_info_MOD.js (hunk 30 above)
Comment (tlfn) [reply actions/store_user_info_MOD.js:165]: So does this

File: actions/store_game_info_MOD.js (hunk 29 above)
Comment (tlfn) [reply actions/store_game_info_MOD.js:150]: So does this

File: actions/remove_message_reactions_MOD.js (hunk 28 above)
Comment (tlfn) [reply actions/remove_message_reactions_MOD.js:45]: so does this

File: actions/control_permissons_MOD.js (hunk 17 above)
Comment (tlfn) [reply actions/control_permissons_MOD.js:168]: so does this

File: actions/control_permissons_MOD.js (hunk 16 above)
Comment (tlfn) [reply actions/control_permissons_MOD.js:167]: so does this

File: actions/control_permissons_MOD.js (hunk 15 above)
Comment (tlfn) [reply actions/control_permissons_MOD.js:163]: so does this

File: actions/control_permissons_MOD.js (hunk 14 above)
Comment (tlfn) [reply actions/control_permissons_MOD.js:161]: so does this

File: actions/control_permissons_MOD.js (hunk 13 above)
Comment (tlfn) [reply actions/control_permissons_MOD.js:160]: so does this

File: actions/control_permissons_MOD.js (hunk 12 above)
Comment (tlfn) [reply actions/control_permissons_MOD.js:156]: so does this

File: actions/control_permissons_MOD.js (hunk 11 above)
Comment (tlfn) [reply actions/control_permissons_MOD.js:154]: so does this

File: actions/control_permissons_MOD.js (hunk 10 above)
Comment (tlfn) [reply actions/control_permissons_MOD.js:153]: so does this

File: actions/control_permissons_MOD.js (hunk 9 above)
Comment (tlfn) [reply actions/control_permissons_MOD.js:149]: so does this

File: actions/check_if_user_reacted_MOD.js (hunk 8 above)
Comment (tlfn) [reply actions/check_if_user_reacted_MOD.js:119]: so does this

File: actions/check_if_user_reacted_MOD.js (hunk 7 above)
Comment (tlfn) [reply actions/check_if_user_reacted_MOD.js:117]: so does this

File: actions/check_if_member_has_role_MOD.js (hunk 6 above)
Comment (tlfn) [reply actions/check_if_member_has_role_MOD.js:121]: so does this

File: actions/check_if_member_has_role_MOD.js (hunk 5 above)
Comment (tlfn) [reply actions/check_if_member_has_role_MOD.js:119]: so does this

**Comment by Fisheiyy** [comment]
just woke up to 40 emails from github about commits maybe i should unsubscribe

File: actions/store_user_info_MOD.js [hunk 33]
```diff
@@ -155,16 +162,12 @@ module.exports = {
...
+        result = user.presence.activities.filter((s) => s.type !== 'CUSTOM_STATUS')
+        result = result?.[0]?.name
         break
       case 8: // User Custom Status
-        if (this.dest(user.presence, 'activities')) {
-          const status = user.presence.activities.filter((s) => this.dest(s, 'type') === 'CUSTOM_STATUS')
-          result = status && this.dest(status[0], 'state')
-        }
+        result = user.presence.activities.filter((s) => s.type === 'CUSTOM_STATUS')
+        result = result?.[0]?.state
```
Comment (almeidx) [inline actions/store_user_info_MOD.js:170]:
```suggestion
        result = user.presence.activities.find((s) => s.type === 'CUSTOM_STATUS')?.[0]?.state
```

File: actions/store_user_info_MOD.js [hunk 34]
```diff
@@ -155,16 +162,12 @@ module.exports = {
         result = user.lastMessageID
         break
       case 7: // User Activities
-        if (this.dest(user.presence, 'activities')) {
-          const status = user.presence.activities.filter((s) => s.type !== 'CUSTOM_STATUS')
-          result = status && this.dest(status[0], 'name')
-        }
+        result = user.presence.activities.filter((s) => s.type !== 'CUSTOM_STATUS')
+        result = result?.[0]?.name
```
Comment (almeidx) [inline actions/store_user_info_MOD.js:166]:
```suggestion
        result = user.presence.activities.find((s) => s.type !== 'CUSTOM_STATUS')?.[0]?.name
```

**Review by almeidx — CHANGES_REQUESTED** [review]

File: actions/check_if_member_has_role_MOD.js [hunk 35]
```diff
@@ -116,15 +116,9 @@ module.exports = {
...
-        result = member.every(function (mem) {
-          if (this.dest(mem, 'roles', 'cache')) {
-            return mem.roles.cache.has(role.id)
-          } else {
-            return false
-          }
-        })
-      } else if (this.dest(member, 'roles', 'cache')) {
-        result = member.roles.cache.has(role.id)
+        result = member.every((mem) => mem?.roles?.cache.has(role.id))
```
Comment (almeidx) [inline actions/check_if_member_has_role_MOD.js:119]:
```suggestion
        result = member.every((mem) => mem?.roles.cache.has(role.id))
```

File: actions/check_if_member_has_role_MOD.js [hunk 36]
```diff
@@ -116,15 +116,9 @@ module.exports = {
...
+      } else {
+        result = !!member?.roles?.cache.has(role.id)
```
Comment (almeidx) [inline actions/check_if_member_has_role_MOD.js:121]:
```suggestion
        result = !!member?.roles.cache.has(role.id)
```

File: actions/check_if_user_reacted_MOD.js [hunk 37]
```diff
@@ -114,13 +114,9 @@ module.exports = {
...
     let result
     if (Array.isArray(member)) {
-      result = member.every(user => {
-        if (this.dest(reaction, 'users', 'cache') && user) {
-          return reaction.users.cache.has(user.id)
-        }
-      })
-    } else if (this.dest(reaction, 'users', 'cache') && member) {
-      result = reaction.users.cache.has(member.id)
+      result = member.every((user) => reaction?.users?.cache.has(user.id))
```
Comment (almeidx) [inline actions/check_if_user_reacted_MOD.js:117]:
```suggestion
      result = member.every((user) => reaction?.users.cache.has(user.id))
```

File: actions/check_if_user_reacted_MOD.js [hunk 38]
```diff
@@ -114,13 +114,9 @@ module.exports = {
...
+    } else {
+      result = !!reaction?.users?.cache.has(member.id)
```
Comment (almeidx) [inline actions/check_if_user_reacted_MOD.js:119]:
```suggestion
      result = !!reaction?.users.cache.has(member.id)
```

**Review by almeidx — CHANGES_REQUESTED** [review]

**Review by almeidx — APPROVED** [review]

**Pull request closed without merging**